tools/cli/src/cli/lib.rs

```diff
@@ -10,12 +10,23 @@
 #![deny(clippy::trivially_copy_pass_by_ref)]
 // We allow expect since it forces good error messages at the least.
 #![allow(clippy::expect_used)]
-
 #[macro_use]
 extern crate tracing;
 
 use crate::common::OpType;
-use std::path::PathBuf;
+use cursive::{
```

Comment:
For now I'd like this to be a separate feature and in it's own file. 

What about a simple text-only version like the credential reset dialogue? Not every terminal can do tui.

Reply:
I actually started doing it that way, but i ended up with many rough edges, like no closable pop ups, no countdown, and something else I can’t remember now. Still doable though. I still have that version and it’s like 80% complete, should I add it somewhere?

Reply:
As nice as all the features are, the user just needs to get prompted with the token and told they've got x minutes to complete it, and success/errors if it fails 😄

The prettiness can happen in the web UI which has *much* better native handling for such things.

Reply:
Alright then :) I think I got everything working, I'll push shortly.